src/healthcheck/HealthChecker.ts

```diff
@@ -286,7 +286,11 @@ class Plugin {
         })
         .catch((error) => {
```

Comment:
not your code, but this is non-idiomatic, `err` is the customary name for this property.